hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/signature/AuthorizationV4HeaderParser.java

```diff
@@ -189,7 +191,14 @@ private Credential parseCredentials(String credential)
 
     // Date should not be empty and within valid range.
     if (!credentialObj.getDate().isEmpty()) {
-      validateDateRange(credentialObj);
+      try {
+        validateDateRange(credentialObj);
+      } catch (DateTimeParseException ex) {
+        throw new MalformedResourceException(
+                "AWS date format is invalid. credential:" + credential,
+                authHeader);
```

Comment:
indentation

```suggestion
        throw new MalformedResourceException(
            "AWS date format is invalid. credential:" + credential, authHeader);
```

You can avoid this in the future by importing ozone code style:

https://github.com/apache/ozone/blob/dd003040a41def491e8de003ef8539ce40854972/CONTRIBUTING.md#L115-L121